besluit/src/test/kotlin/com/ritense/besluit/BaseIntegrationTest.kt

```diff
@@ -78,9 +73,6 @@ class BaseIntegrationTest : BaseTest() {
     @Autowired
     lateinit var connectorTypeInstanceRepository: ConnectorTypeInstanceRepository
 
-    @MockBean
-    lateinit var mailSender: MailSender
```

Comment:
While the bean was not used in the code, the mock prevents the application from using a real mailsender during tests.

Reply:
I replaced the MailSender MockBean with this dependency:
```testImplementation project(":mail:local-mail")```
Which does nothing except for logging